packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -107,6 +108,7 @@ export function importGlobPlugin(config: ResolvedConfig): Plugin {
 }
 
 const importGlobRE = /\bimport\.meta\.glob(?:<\w+>)?\s*\(/g
+const objectKeysRE = /Object\.keys\(\s*$/
```

Comment:
```suggestion
const objectKeysRE = /\bObject\.keys\(\s*$/
```
I guess we need `\b` here to avoid matching other things like `aObject.keys`.